teaser/data/input/teaserjson_input.py

```diff
@@ -269,6 +297,42 @@ def load_teaser_json(path, project):
                 ceil.name = cl_name
                 set_basic_data_teaser(cl_in, ceil)
                 set_layer_data_teaser(cl_in, ceil)
+            try:
```

Comment:
I have to admit that I do not really like the the try/except approach here. I would suggest to use `get` functionality to retrieve default values if the key could not be found. What do you think about this? If you agree please change this also at the other occurenes in this function

Reply:
Thanks for adding the changes and I'm happy that I could help :)